src/tests/PartyTest.js

```diff
@@ -101,4 +111,13 @@ describe('Test Party Endpoints', () => {
         done();
       });
   });
+  it('Should return the array of all political parties', (done) => {
+    chai.request(app)
+      .get('/api/v1/parties')
+      .end((err, res) => {
+        expect(res).to.have.status(200);
+        expect(res.body.data).to.have.lengthOf(1);
+        done();
```

Comment:
Please assert for some of the response in the array returned

Reply:
Okay. I'll do that.